### DISCUSSION TITLE
[Design] : Can the Logo be a bit modified ?

### DISCUSSION THREAD

**Discussion by the-rough-page** [apache/skywalking#8763] [Q&A] [answered]
If I want to add something to the logo What is the language it is written in and how can we do it is there any solution for this?

**Comment by wu-sheng** [comment, marked as answer]
This is just a component-logo mapping in the UI. 

All icons are here, https://github.com/apache/skywalking-rocketbot-ui/tree/master/src/views/components/topology/assets. If you want to replace logos for different components
